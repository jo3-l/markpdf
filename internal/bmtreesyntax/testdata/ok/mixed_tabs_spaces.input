foo, 1
    fgh, 3
    	jkl , 4