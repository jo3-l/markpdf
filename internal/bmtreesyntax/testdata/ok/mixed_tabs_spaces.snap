- "foo" @ p1
	- "fgh" @ p3
		- "jkl " @ p4